test/e2e_node/jenkins/image-config-serial.yaml

```diff
@@ -12,15 +12,15 @@ images:
   containervm:
     image: e2e-node-containervm-v20161208-image # docker 1.11.2
     project: kubernetes-node-e2e-images
-  cos-stable:
```

Comment:
This is to match the channel-based test naming by eliminating the idea of the cos specific milestone names. We use "stable 2" in other tests as well (https://k8s-testgrid.appspot.com/cos-image-validation).